.github/workflows/release.yml

```diff
@@ -16,10 +16,10 @@ jobs:
     steps:
       - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # renovate: tag=v2
         with:
-          # pulls all commits (needed for lerna to correctly version)
+          # pulls all commits (needed for computing the next version)
           # see https://stackoverflow.com/a/60184319/9285308 & https://github.com/actions/checkout
```

Comment:
The stack overflow link might not be relevant anymore then.

Reply:
We still fetch the commits and tags to determine the next version, so still relevant